src/api/app/views/webui2/shared/_download_repository_link.html.haml

```diff
@@ -0,0 +1,4 @@
+- url = "#{download_url}/#{project.to_s.gsub(/:/, ':/')}/#{repository}"
```

Comment:
I don't think you need `to_s.gsub(/:/, ':/')` For example, if you go to https://download.opensuse.org/repositories/OBS:Server:2.9:Staging/SLE_12_SP3 it works, although it redirect you. So there is no need to complicate the code.

Reply:
it works, because it get redirected. 

```
vpereira@kimura:~> curl https://download.opensuse.org/repositories/OBS:Server:2.9:Staging/SLE_12_SP3
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://download.opensuse.org/repositories/OBS:Server:2.9:/Staging/SLE_12_SP3">here</a>.</p>
<hr>
<address>Apache/2.4.23 (Linux/SUSE) Server at download.opensuse.org Port 443</address>
</body></html>
```

The card is about migrating code to bootstrap, not changing stuff that we don't know. I would prefer to don't touch it, but for sure I can take two weeks refactoring the ```PackageController#binaries```

Reply:
Agree with @vpereira it was like this before so leave it for now.

Reply:
ok :+1: